open_spiel/games/CMakeLists.txt

```diff
@@ -192,18 +192,21 @@ if (${OPEN_SPIEL_BUILD_WITH_ACPC})
   set(GAME_SOURCES ${GAME_SOURCES} universal_poker/universal_poker.cc universal_poker/universal_poker.h)
 endif()
 
-add_library (games OBJECT ${GAME_SOURCES})
-
+add_library (games SHARED ${GAME_SOURCES})
```

Comment:
Are you sure this won't enforce the change to all binaries built within OpenSpiel, i.e. not just keep the dynamic linking local to the tests?

Reply:
Oh wait, no.. because they still link to them using TARGET_OBJECTS: https://github.com/google-deepmind/open_spiel/blob/a95a9d6589a57e9c7587aa228785aae16f7597f3/open_spiel/CMakeLists.txt#L223

Ok!